src/database-gdbm.cpp

```diff
@@ -30,6 +30,9 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "porting.h"
 #include "util/string.h"
 
+// Change this IF you change serialization way to match width needed to serialize V3S16 vector
+#define GDBM_KEY_WIDTH 6
```

Comment:
this will always be `3 * sizeof(s16)` aka `6`, doesn't really make sense to have a #define for this